template.yaml

```diff
@@ -123,6 +123,16 @@ parameters:
 - name: DISABLE_QUERY_SYSTEM_PROMPT
   value: "true"
   description: "Corresponds to the lightspeed config customization.disable_query_system_prompt"
+- name: LIGHTSPEED_STACK_POSTGRES_HOST
+  description: "Postgres host for the lightspeed stack database"
+- name: LIGHTSPEED_STACK_POSTGRES_PORT
+  description: "Postgres port for the lightspeed stack database"
+- name: LIGHTSPEED_STACK_POSTGRES_NAME
+  description: "Postgres database name for the lightspeed stack"
+- name: LIGHTSPEED_STACK_POSTGRES_USER
+  description: "Postgres user for the lightspeed stack database"
+- name: LIGHTSPEED_STACK_POSTGRES_PASSWORD
+  description: "Postgres password for the lightspeed stack database"
```

Comment:
How are these being set when we deploy to int? Do we need to set them in app-interface?